hello_git.txt

```diff
@@ -1,2 +1,2 @@
 Hello, we are learning git.
-
+We also learned forking, cloning and branching. 
```

Comment:
Thanks @tofaaan for adding this text. Yea we are really having fun learning git

Reply:
You are really very welcome. Could you please merge